psycopg_pool/psycopg_pool/_acompat.py

```diff
@@ -0,0 +1,162 @@
+"""
+Utilities to ease the differences between async and sync code.
+
+These object offer a similar interface between sync and async versions; the
+script async_to_sync.py will replace the async names with the sync names
+when generating the sync version.
+"""
```

Comment:
This file seems to have a lot of overlap with `psycopg/psycopg/_acompat.py`
Is there any possibility to share code between the two packages?

Reply:
Yes, it's to keep them isolated from each other: there isn't a 1:1 version lock between pool and psycopg. 

If the pool used `_acompat` objects from psycopg it would become dependent on whatever version we would distribute those objects, and they would become a psycopg public interface that we shouldn't break.

We can afford to do that between psycopg and psycopg-c, because they have a 1:1 release pattern and each minor version is only expected to depend on the same minor version of the other package, but for the pool we decided on an release cycle independent from the main package.